manifests/install/server/master.pp

```diff
@@ -68,7 +68,7 @@
   -> exec { "server_install_${$facts['fqdn']}":
     command     => $server_install_cmd,
     environment => [ "IPA_ADMIN_PASS=${admin_pass}", "DS_PASSWORD=${ds_password}" ],
-    path        => ['bin', '/sbin', '/usr/sbin'],
+    path        => ['/bin', '/sbin', '/usr/sbin', '/usr/bin'],
```

Comment:
FYI needed to add this because the path to systemctl is /usr/bin in CentOS 8
```
[root@freeipa ~]# which systemctl
/usr/bin/systemctl
```

Reply:
lol